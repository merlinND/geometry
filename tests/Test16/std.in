L myLine 12
L myLine 10 0 1 2
EXIT
