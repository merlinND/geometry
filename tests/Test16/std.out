L myLine 2 7 2 7
OK
